lib/config/presets/internal/monorepo.ts

```diff
@@ -2,6 +2,7 @@ import is from '@sindresorhus/is';
 import type { Preset } from '../types';
 
 const repoGroups = {
+  acot: 'https://github.com/acot-a11y/acot',
```

Comment:
```suggestion
  'acot': 'https://github.com/acot-a11y/acot',
```

Reply:
nope, only quote if required

Reply:
Hi @james-d-elliott, Thank you for your review.

Object keys can be `-` or `. `, otherwise it will be formatted by prettier.
Should I add a comment to intentionally ignore?

Reply:
leave as is without quotes.

Reply:
Okay no worries :) Just noticed it in passing!